pom.xml

```diff
@@ -397,9 +403,9 @@
               <links>
                 <link>http://download.oracle.com/javase/7/docs/api/</link>
                 <link>http://cloud.google.com/appengine/docs/java/javadoc</link>
-                <link>https://jar-download.com/artifacts/org.codehaus.jackson/jackson-core-asl/${project.jackson-core-asl.version}/documentation</link>
+                <link>https://jar-download.com/javaDoc/org.codehaus.jackson/jackson-core-asl/${project.jackson-core-asl.version}/documentation</link>
```

Comment:
I think the correct link is without the /documentation - that returns javadoc for me

Reply:
Good call. Fixed